### PR TITLE
[Javascript] Adding Moleculer

### PR DESCRIPTION
**Pull request by AndreMaz**
Adding Moleculer https://github.com/the-benchmarker/web-frameworks/issues/1160

### CODE REVIEW

**Comment by waghanza** [comment]
Thanks @AndreMaz

Glad to review and other `node` based framework

File: neph.yaml [hunk 1]
```diff
@@ -0,0 +1,483 @@
+---
```
[the remark this answers is not in the document]
Comment (AndreMaz) [reply neph.yaml]: Fixed

**Comment by AndreMaz** [comment]
@waghanza all tests are green. Can you review it again? Is everything ok?

**Comment by waghanza** [comment]
@AndreMaz everything seems ok. Let me get back behind a computer for final tests and merge :tada:

**Comment by AndreMaz** [comment]
Awesome 😁 thanks

**Pull request merged**